medical_pathology/demo/medical_pathology_demo.xml

```diff
@@ -22,4 +22,44 @@
                eval="[(6, 0, [ref('medical_pathology.medical_pathology_group_2')])]"></field>
     </record>
 
+    <record id="medical_pathology_medical_pathology_A00" model="medical.pathology">
+        <field name="name">Cholera</field>
+        <field name="code">A00</field>
+        <field name="code_type_id" ref="medical_pathology.pathology_code_05" />
+        <field name="category_id" ref="medical_pathology.medical_pathology_category_7" />
+    </record>
+
+    <record id="medical_pathology_medical_pathology_A00_0" model="medical.pathology">
+        <field name="name">Cholera due to Vibrio cholerae 01, biovar cholerae</field>
+        <field name="code">A00.0</field>
```

Comment:
These ICD10 codes will conflict when data is imported in #134. Add an underscore or something to create a unique namespace while providing somewhat relevant data for demo